x-pack/plugins/observability/public/pages/slo_details/components/header_control.tsx

```diff
@@ -136,31 +185,58 @@ export function HeaderControl({ isLoading, slo }: Props) {
             <EuiContextMenuItem
               key="manageRules"
               disabled={!hasWriteCapabilities}
+              icon="gear"
               onClick={handleNavigateToRules}
               data-test-subj="sloDetailsHeaderControlPopoverManageRules"
             >
               {i18n.translate('xpack.observability.slo.sloDetails.headerControl.manageRules', {
                 defaultMessage: 'Manage rules',
               })}
             </EuiContextMenuItem>,
-          ].concat(
-            !!slo && isApmIndicatorType(slo.indicator.type)
-              ? [
-                  <EuiContextMenuItem
-                    key="exploreInApm"
-                    onClick={handleNavigateToApm}
-                    data-test-subj="sloDetailsHeaderControlPopoverExploreInApm"
-                  >
-                    {i18n.translate(
-                      'xpack.observability.slos.sloDetails.headerControl.exploreInApm',
-                      {
-                        defaultMessage: 'Explore in APM',
-                      }
-                    )}
-                  </EuiContextMenuItem>,
-                ]
-              : []
-          )}
+          ]
+            .concat(
+              !!slo && isApmIndicatorType(slo.indicator.type)
+                ? [
+                    <EuiContextMenuItem
+                      key="exploreInApm"
+                      icon="bullseye"
+                      onClick={handleNavigateToApm}
+                      data-test-subj="sloDetailsHeaderControlPopoverExploreInApm"
+                    >
+                      {i18n.translate(
+                        'xpack.observability.slos.sloDetails.headerControl.exploreInApm',
+                        {
+                          defaultMessage: 'Service details',
+                        }
+                      )}
+                    </EuiContextMenuItem>,
+                  ]
+                : []
```

Comment:
Not related to your PR but could you remove the unecessary wrapping array around the menu item please?
```suggestion
                ? 
                    <EuiContextMenuItem
                      key="exploreInApm"
                      icon="bullseye"
                      onClick={handleNavigateToApm}
                      data-test-subj="sloDetailsHeaderControlPopoverExploreInApm"
                    >
                      {i18n.translate(
                        'xpack.observability.slos.sloDetails.headerControl.exploreInApm',
                        {
                          defaultMessage: 'Service details',
                        }
                      )}
                    </EuiContextMenuItem>
                  
                : []
```